packages/kit/src/runtime/server/page/respond.js

```diff
@@ -19,15 +19,30 @@ import { coalesce_to_error } from '../../../utils/error.js';
  *   $session: any;
  *   route: import('types/internal').SSRPage;
  *   page: import('types/page').Page;
+ *   resolve_params: import('types/hooks').RequiredResolveParams;
  * }} opts
  * @returns {Promise<ServerResponse | undefined>}
  */
 export async function respond(opts) {
-	const { request, options, state, $session, route } = opts;
+	const { request, options, state, $session, route, resolve_params } = opts;
 
 	/** @type {Array<SSRNode | undefined>} */
 	let nodes;
 
+	if (resolve_params.allowSSR) {
```

Comment:
Do you mean `!resolve_params.allowSSR` here?

Reply:
Yes! good catch